babybuddy/tests/tests_forms.py

```diff
@@ -30,6 +30,7 @@ def setUpClass(cls):
             'last_name': 'Name',
             'email': 'user@user.user',
             'dashboard_refresh_rate': '',
+            'dashboard_hide_empty': 'off',
```

Comment:
FYI there have been some changes to the template settings here in recent commits. You may want to rebase once more just to make sure there are no conflicts. I don't think there will be any,